### PR TITLE
Move AnalyzedField context from RawReport to ReportColumn

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

File: lib/BenchShow/Common.hs [hunk 1]
```diff
@@ -1084,15 +1084,14 @@ data ReportColumn = ReportColumn
...
     } deriving Show
 
--- XXX put reportAnalyzed in reportColumns
 data RawReport = RawReport
     { reportOutputFile :: Maybe FilePath
     , reportIdentifier :: String
     , reportRowIds     :: [String]
     , reportColumns    :: [ReportColumn]
-    , reportAnalyzed   :: [[AnalyzedField]]
     , reportEstimators :: Maybe [[Estimator]]
```
Comment (adithyaov) [inline lib/BenchShow/Common.hs:1095]: I can maybe move `reportEstimators` into `ReportColumn` as well? Make it more localized.

File: .github/workflows/packcheck.yml [hunk 2]
```diff
@@ -95,38 +95,44 @@ jobs:
...
-          #   runner: ubuntu-latest
-
-          # - name: ci
-          #   ghc_version: 8.8.3
-          #   command: cabal-v2
-          #   runner: macos-latest
+        name:
+          - 8.4.4
+          - 8.6.5
+          - 8.8.4
```
Comment (harendra-kumar) [inline .github/workflows/packcheck.yml:101]: 8.10 is the prime version right now. We should add that.

**Review by harendra-kumar — APPROVED** [review]
You can change the bounds of base, Chart, and Chart-diagrams to use the second major version component too.

**Comment by adithyaov** [comment]
Chart, and Chart-diagrams, look fine. The latest is 1.9.3 so an upper bound of 2 looks good.

**Pull request merged**